elasticdl/python/master/tensorboard_service.py

```diff
@@ -29,6 +29,8 @@ def write_dict_to_summary(self, dictionary, version):
             for k, v in dictionary.items():
                 if isinstance(v, np.ndarray) and len(v) == 1:
                     v = v[0]
+                elif isinstance(v, dict) and len(v):
+                    v = list(v.values())[0]
```

Comment:
In deepFM test, the dictionary is as following:

```
dict_keys(['logits', 'probs'])
[2019-11-19 04:05:57,638] [INFO] [evaluation_service.py:255:complete_task] {'logits': {'accuracy': <tf.Tensor: id=4011, shape=(), dtype=float32, numpy=0.66985446>}, 'probs': {'auc': <tf.Tensor: id=4044, shape=(), dtype=float32, numpy=0.5908479>}}
```

We have to handle such case.

Reply:
Done